Helios/Windows/Controls/About.xaml.cs

```diff
@@ -27,7 +27,7 @@ namespace GadrocsWorkshop.Helios.Windows.Controls
     public partial class About : Window
     {
         // these are in order of being primary on the project
-        public static readonly string[] Authors = { "Gadroc", "BlueFinBima", "derammo" };
+        public static readonly string[] Authors = { "Gadroc", "derammo", "BlueFinBima" };
```

Comment:
I put the authors in historical order, to denote the life cycle phases of Helios :)

Reply:
appreciate the nod though!

Reply:
My preference is in contribution order - hence the acknowledgement to all that you have done.